### DISCUSSION TITLE
how to create two wallet in one node

### DISCUSSION THREAD

**Discussion by yuzhongrong** [Chia-Network/chia-blockchain#5268] [Support]
how to create two wallet in one node

**Comment by yuzhongrong** [comment]
What command to use?

**Reply by pluto2021** [reply]
Same question. We need a 2nd wallet to be able to sync at a same time.  You can get a 2nd wallet key on the main mode, just close the GUI and reopen it. You can choose to generate a new wallet. But you can only sync one at a time using one ISP modem connection due to port forwarding issue.  Not sure which default key would be used to plot using the CLI though. Strangely, under the new key, the old plots are still recognized as valid plots if you log in using the new private key!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
